frontend-pwa/src/app/App.tsx

```
@@ -0,0 +1,46 @@
+import { listUsers } from '../api/client';
+
+export function App() {
+  const { data, isLoading, isError, error } = useQuery({
+    queryKey: ['users'],
+    queryFn: listUsers,
+    staleTime: 60_000,
+  });
+
+  if (isLoading) {
+    return (
+      <div className="p-6 min-h-screen bg-base-200 text-base-content" role="status">
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<output>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)